src/ImageSharp.Web/Resolvers/PhysicalFileSystemResolver.cs

```diff
@@ -12,21 +12,15 @@ namespace SixLabors.ImageSharp.Web.Resolvers
     public class PhysicalFileSystemResolver : IImageResolver
     {
         private readonly FileInfo fileInfo;
-        private readonly ImageMetadata metadata;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="PhysicalFileSystemResolver"/> class.
         /// </summary>
         /// <param name="fileInfo">The input file info.</param>
-        /// <param name="metadata">The image metadata associated with this file.</param>
-        public PhysicalFileSystemResolver(FileInfo fileInfo, in ImageMetadata metadata)
```

Comment:
See how the struct was passed via `in` here? We're passing a ref to the readonly struct and avoiding recreating it each time.